README.md

```diff
@@ -73,14 +73,15 @@ expect that the project is already built.
 ### Scoping
 
 Many commands (notably `build`, `dev`, `test`) support package scoping to
-save development time. To rebuild only `xod-cli` and its dependencies:
+save development time. To rebuild only `xod-cli`:
 
-    $ yarn build -- xod-cli
+    $ yarn build --scope xod-cli
 
-To rebuild only `xod-cli` without dependencies:
+To rebuild `xod-cli` and its dependencies:
 
-    $ yarn build -- xod-cli --only
+    $ yarn build --scope xod-cli --include-filtered-dependencies
 
+Those are standard [Lerna](https://github.com/lerna/lerna) flags, you can learn more about them [here](https://github.com/lerna/lerna#flags).
```

Comment:
Please, don’t use words like here/there/link for link anchors. Link destination should be evident even if surrounding text would be hidden:

```
Those are standard [Lerna flags](https://github.com/lerna/lerna#flags).
```

…that’s all